Packs/Phishing/ReleaseNotes/3_1_6.md

```diff
@@ -0,0 +1,4 @@
+
+#### Playbooks
+##### Process Microsoft's Anti-Spam Headers
+- The **Set BCL score** task was updated to extract the correct BCL scores (4-9) on the second transformer.
```

Comment:
I'm more in favor of making it easier for the customer to understand the meaning of that fix. Consider something like: "Fixed an issue where the BCL (Bulk Complaint Level) was not extracted correctly from the email headers."